### PR TITLE
Updated the Gradle Dependencies

### PR DESCRIPTION
**Pull request by RaghavAwasthi**
This Pull request safely upgrades the gradle dependencies 
I have tested the code before and after their is no issue . PR can be safely merged

### CODE REVIEW

**Comment by RaghavAwasthi** [comment]
@the-dagger  With reference to this PR https://github.com/JBossOutreach/lead-management-android/pull/148  It was all messed up This is a clean version of the PR apt with the guidlines of the task . Doing nothing less nothing more as you wanted it to be. 
Can you merge it now

**Comment by the-dagger** [comment]
Sounds good thanks!
We will merge the Pull requests later, but thank you for this :)

File: app/build.gradle [hunk 1]
```diff
@@ -30,63 +30,51 @@ android {
         targetCompatibility 1.8
         sourceCompatibility 1.8
     }
-    buildToolsVersion '27.0.3'
+    buildToolsVersion '28.0.3'
```
Comment (sidhyatikku) [inline app/build.gradle]: @RaghavAwasthi also update the `build-tools` & `android` in `.travis.yml`

Comment (RaghavAwasthi) [reply app/build.gradle]: @sidhyatikku I have made the changes you requested

**Pull request merged**